修正: ニコ生エリアのエラーダイアログを英訳

yushinakatani opened the pull request:
# このpull requestが解決する内容
言語を英語に選択していても日本語のエラーダイアログが表示される問題

# 関連するIssue（あれば）
#451

yushinakatani committed:
ニコ生エリアのエラーダイアログ英訳対応

koizuka commented on app/i18n/en-US/nicolive-program.json:
```diff
@@ -127,80 +127,80 @@
-          "title": "ログインしていません",
-          "message": "ログインしていないため、放送者NG設定を削除できませんでした。\nN Airメイン画面右上のログインボタンからniconicoにログインしてください。"
+          "title": "You are not logged in",
+          "message": "Block settings can not be deleted because you are not logged in to niconico on N Air.\nLog in to niconico from the login button on the upper right of the N Air main screen."
```
他は `The Block settings` だけど、ここは `the` がないのは意図したものだろうか

yushinakatani replied:
意図ではなくミスでした。修正いたしましたので最後ご確認をお願いします。

yushinakatani committed:
冠詞とそのほかの軽微な修正

・block settingsに冠詞がある場合と無い場合があったが、ボタンなどのUIではなくエラーメッセージ中なので冠詞を付ける
・program informationに冠詞がある場合と無い場合があったが、冠詞ありに揃えた
・文中でもblock settingsの頭文字が大文字になっていたがUIのボタンを指している訳では無く機能そのものを指定しているので小文字に修正した

koizuka commented on app/i18n/en-US/nicolive-program.json:
```diff
@@ -127,80 +127,80 @@
         "400": {
-          "title": "放送者NG設定を追加できませんでした",
-          "message": "放送者NG設定を追加できませんでした。\n%{additionalMessage}"
+          "title": "The Block settings could not be added",
```
```suggestion
          "title": "The block settings could not be added",
```

koizuka commented on app/i18n/en-US/nicolive-program.json:
```diff
@@ -127,80 +127,80 @@
-          "title": "放送者NG設定を追加できませんでした",
-          "message": "放送者NG設定を追加できませんでした。\n%{additionalMessage}"
+          "title": "The block settings could not be added",
+          "message": "The Block settings could not be added\n%{additionalMessage}"
```
```suggestion
          "message": "The block settings could not be added\n%{additionalMessage}"
```

koizuka commented on app/i18n/en-US/nicolive-program.json:
```diff
@@ -127,80 +127,80 @@
-          "title": "放送者NG設定を削除する権限がありません",
-          "message": "ログインしているアカウントに放送者NG設定を削除する権限がないため、番組時間を延長できませんでした。\n正しいアカウントでログインしているかを確認してください。"
+          "title": "You do not have the authority to delete the Block settings",
+          "message": "You can not load the block settings because the account you are logged into does not have authority to load the Block settings.\nMake sure you are logged in with the correct account."
```
```suggestion
          "message": "You can not delete the block settings because the account you are logged into does not have authority to load the Block settings.\nMake sure you are logged in with the correct account."
```

koizuka reviewed (commented):
細かい修正

yushinakatani committed:
Update app/i18n/en-US/nicolive-program.json

Co-authored-by: Akihiko Koizuka <koizuka@gmail.com>

yushinakatani committed:
Update app/i18n/en-US/nicolive-program.json

Co-authored-by: Akihiko Koizuka <koizuka@gmail.com>

yushinakatani committed:
Update app/i18n/en-US/nicolive-program.json

Co-authored-by: Akihiko Koizuka <koizuka@gmail.com>
